MAUI/DataForm/migration.md

```diff
@@ -628,6 +687,81 @@ The following properties of the `Xamarin SfDataForm` is moved to the [DataFormDe
 </tr>
 </table>
 
+#### TextInputLayoutSettings
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ContainerType](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.TextInputLayoutSettings.html#Syncfusion_XForms_DataForm_TextInputLayoutSettings_ContainerType)'| markdownify }}</td>
+<td>ContainerType</td>
+<td>Gets or sets a value that determines the container types for floating label layout.</td>
+</tr>
+
+<tr>
+<td>{{'[OutlineCornerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.TextInputLayoutSettings.html#Syncfusion_XForms_DataForm_TextInputLayoutSettings_OutlineCornerRadius)'| markdownify }}</td>
+<td>OutlineCornerRadius</td>
+<td>Gets or sets a value to customize the customize the corner radius of outline border in the floating label layout.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>EnableFloating</td>
+<td>Gets or sets a value that indicates whether to float the label when it is focused or unfocused.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>IsHintAlwaysFloated</td>
+<td>Gets or sets a value that indicates whether to fix the hint label always at the top even when the text is empty in the floating label layout.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>EnableHintAnimation</td>
+<td>Gets or sets a value that indicates whether to enable animation for the hint text when the input view is focused or unfocused in the floating label layout.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>HelperTextStyle</td>
+<td>Gets or sets the style for helper label in the floating label layout.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>Stroke</td>
+<td>Gets or sets the border  color based on the container in the floating label layout. </td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>FocussedStrokeThickness</td>
+<td>Gets or sets as value to customize the stroke width in focused state in the floating label layout, it is applicable for the bottom line and outline border. when setting the container type as filled and outlined respectively.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>UnfocussedStrokeThickness</td>
+<td>Gets or sets as value to customize the stroke width in unfocused state in the floating label layout, it is applicable for the bottom line and outline border when setting the container type as filled and outlined respectively.</td>
+</tr>
+
+<tr>
+<td>{{'[ShowHelperText](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.TextInputLayoutSettings.html#Syncfusion_XForms_DataForm_TextInputLayoutSettings_ShowHelperText)'| markdownify }}</td>
+<td>ShowHelperText</td>
+<td>Gets or sets a value indicating whether to display the helper text and error text. It determines the visibility of the helper text and error text.</td>
```

Comment:
<td>Gets or sets a value indicating whether to display the helper and error text. It determines the visibility of the helper text and error text.</td>

Reply:
Content changed